yuniql-core/ConnectionExtensions.cs

```diff
@@ -135,25 +135,10 @@ public static IDbConnection KeepOpen(this IDbConnection connection)
             IDbTransaction transaction = null,
             ITraceService traceService = null)
         {
-            var stopwatch = new Stopwatch();
-            stopwatch.Start();
-            var statementCorrelationId = Guid.NewGuid().ToString().Fixed();
-            traceService?.Debug($"Executing statement {statementCorrelationId}: {Environment.NewLine}{commandText}");
-
-            var command = connection
-                .KeepOpen()
-                .CreateCommand(
-                commandText: commandText,
-                commandTimeout: commandTimeout,
-                transaction: transaction);
-
+            var output = connection.QuerySingle(commandText, commandTimeout: commandTimeout, transaction: transaction, traceService: traceService);
```

Comment:
Thanks!

Reply:
Happy to help, its just that i will be keep doing stuff in this pull request. I guess, i should better send this kinda stuff in separate issue/pr.